LiteCore/Database/DatabaseImpl.cc

```diff
@@ -1025,17 +1022,17 @@ namespace litecore {
 #pragma mark - REPLICATION:
 
 
-    static const char * kRemoteDBURLsDoc = "remotes";
+    static const char *kRemoteDBURLsDoc = "remotes";
 
 
     C4RemoteID DatabaseImpl::getRemoteDBID(slice remoteAddress, bool canCreate) {
         bool inTransaction = false;
         bool commit = false;
         DEFER {
-            if (inTransaction) {
-                endTransaction(commit);
-            }
-        };
+                  if (inTransaction) {
```

Comment:
This spacing is weird! Especially that the closing brace is indented farther than the opening line. I suspect the DEFER macro confused the formatter.

Reply:
Fixed with move to Clang-Format